### ISSUE TITLE
Invalid JSON Error: SignInWithEmailAndPasswordAsync

### ISSUE THREAD

**Issue by tnishiki** [step-up-labs/firebase-authentication-dotnet#148]
Hello there,

I'm trying to use SignInWithEmailAndPasswordAsync on FirebaseAuthentication.net 3.5.0, I always got a JSON error.

error message:
```
Invalid JSON payload received. Expected , or } after key:value pair.
****\",\"tenantId\":\"\"\"returnSecureToken\":\n
```
I saw FirebaseAuthProvider.cs changes in [Add multi-tenancy to sign in with email and password #113](https://github.com/step-up-labs/firebase-authentication-dotnet/commit/fc2d8117213f768a64b360829eedd9edc702c6ce).

```cs
- var content = $"{{\"email\":\"{email}\",\"password\":\"{password}\",\"returnSecureToken\":true}}";
+ StringBuilder sb = new StringBuilder($"{{\"email\":\"{email}\",\"password\":\"{password}\",");
+
+if (tenantId != null)
+{
+    sb.Append($"\"tenantId\":\"{tenantId}\"");
+}
+
+sb.Append("\"returnSecureToken\":true}}");
```

If I could fix it, I think it would be:
```cs
StringBuilder sb = new StringBuilder($"{{\"email\":\"{email}\",\"password\":\"{password}\",");

if (tenantId != null)
{
    sb.Append($"\"tenantId\":\"{tenantId}\",");//add a comma
}

sb.Append("\"returnSecureToken\":true}");// delete a brace 
```

Thanks

**Comment by shahtaj-qasim** [comment]
Hi there, I was also getting this same issue. I guess it's the issue with the latest release in FirebaseAuthentication.net (version 3.5.0), so you can just install the previous version that is FirebaseAuthentication.net 3.4.0 and hopefully, it should work :)

**Comment by sl-joao-silva** [comment]
Same problem here. I already solved this issue on a local version that i had (source code ),  but updated to nuget, becuase of code maintenece...
Here is my code that still works
```csharp
public async Task<FirebaseAuthLink> SignInWithEmailAndPasswordAsync(string email, string password
            string tenantId = null)
        {
            String sb = $"{{\"email\":\"{email}\",\"password\":\"{password}\",";

            if (tenantId != null)
            {
                sb+=($"\"tenantId\":\"{tenantId}\"");
            }

            sb+=("\"returnSecureToken\":true}}");
            sb = sb.Replace("{{", "{").Replace("}}", "}"); //added this Line
            return await ExecuteWithPostContentAsync(GooglePasswordUrl, sb.ToString()).ConfigureAwait(false);
        }
```

**Comment by asinino** [comment]
Just to let you know this bug was fixed on PR #136  and was successfully published on Nuget repository with version 3.6.0, you now just need to update your package to the last available version.

Thanks for the fix @sheindel.

@bezysoftware you can now close this issue.

**Comment by tnishiki** [comment]
Thanks for the fix !
